dockstore-webservice/src/main/java/io/dockstore/webservice/Helper.java

```diff
@@ -379,75 +389,67 @@ private static Container findGithubCWL(Container c, Map<String, Repository> gitR
      * @param container
      * @param client
      * @param tokenDAO
-     * @param user
      * @return the updated container
      */
-    private static Container findBitbucketCWL(Container container, HttpClient client, TokenDAO tokenDAO, User user) {
+    private static Container findBitbucketCWL(Container container, HttpClient client, Token token) {
```

Comment:
This class is getting to an epic size. 
For 0.2, let's look together at refactoring this, I see an opportunity to refactor out the source control interactions into an interface with two implementations (github and bitbucket) and I think there's some useless code as well near the end.